src/storm/solver/GurobiLpSolver.h

```diff
@@ -164,15 +142,13 @@ class GurobiLpSolver : public LpSolver<ValueType> {
     // The index of the next variable.
     int nextVariableIndex;
 
-    // The index of the next constraint.
-    int nextConstraintIndex;
-
     // A mapping from variables to their indices.
     std::map<storm::expressions::Variable, int> variableToIndexMap;
 
     struct IncrementalLevel {
         std::vector<storm::expressions::Variable> variables;
         int firstConstraintIndex;
+        int firstGenConstraintIndex;
```

Comment:
What is a GenConstraint?

Reply:
"General Constraint" is a [Gurobi term](https://www.gurobi.com/documentation/current/refman/constraints.html#subsubsection:GeneralConstraints). I'll add a code comment